res/openvpn-gui-res-it.rc

```diff
@@ -236,10 +236,10 @@ FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
     ICON ID_ICO_APP, ID_ICON_ABOUT, 8, 16, 21, 20
-    LTEXT "OpenVPN GUI v%ls - Interfaccia di OpenVPN per Windows\n\
-Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n\
-Copyright (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n\
-Copyright (C) 2012-2022 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
+    LTEXT "OpenVPN GUI v. %ls - Interfaccia OpenVPN per Windows\n\
+    Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n\
+    Copyright (C) 2008-2014 Heiko Hund <heikoh@users.sf.net>\n\
+    Copyright (C) 2012-2022 OpenVPN GUI contributors\n", ID_TXT_VERSION, 36, 15, 206, 50
```

Comment:
Revert these 3 lines to original. Adding space is not correct here -- it will mess up display of these lines.

Reply:
The added space is still there on these lines. See comment above.